frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -24,11 +24,9 @@ import {
 } from '~/types'
 import { Dayjs } from 'dayjs'
 import { preflightLogic } from 'scenes/PreflightCheck/logic'
-import { PersonModalParams } from 'scenes/trends/personsModalLogic'
+import type { PersonModalParams } from 'scenes/trends/personsModalLogic'
```

Comment:
Just trying it out here, but we could add `type` everywhere we only import types, e.g. all logic types. This will tell esbuild to totally ignore these files when bundling, as types are ignored anyway. Kea-typegen should do this automatically.